src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux";
+import Layout from "components/Layout/Layout";
+import { Link } from "react-router-dom";
+import "./GetInvolved.scss";
```

Comment:
Please add the way we are doing this in the main branch now - example: https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/main/src/components/components.scss